python/cudf/cudf/core/index.py

```diff
@@ -523,6 +523,12 @@ def get_slice_bound(self, label, side, kind=None):
         int
             Index of label.
         """
+        if kind is not None:
+            warnings.warn(
+                "'kind' argument in get_slice_bound is deprecated and will be "
+                "removed in a future version.  Do not pass it.",
```

Comment:
I think "Do not pass it" is implied. It sounds kind of heavy handed so maybe we can remove that.
```suggestion
                "removed in a future version.",
```

Reply:
I just blindly copied the pandas warning, but I agree better without.